src/Nethermind/Nethermind.Monitoring/MonitoringService.cs

```diff
@@ -80,6 +83,8 @@ public async Task StartAsync()
                     },
                     OnError = ex =>
                     {
+                        if (ex.InnerException is not SocketException)
```

Comment:
Recommend not logging on `System.Net.Http.HttpRequestException` instead explicitly looking for an exception.
Also, recommend, changing the logger level to if (_logger.IsTrace) instead, instead of skipping it entirely.